src/admin/includes/extra/javascript/rth_stripe.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Stripe integration for modified
+ *
+ * You can find informations about system classes and development at:
+ * https://docs.module-loader.de
+ *
+ * @author  Robin Wieschendorf <mail@robinwieschendorf.de>
+ * @author  Jay Trees <stripe@grandels.email>
+ * @link    https://github.com/RobinTheHood/modified-stripe/
+ */
+
+namespace RobinTheHood\Stripe;
+
+require_once DIR_FS_DOCUMENT_ROOT . '/includes/extra/functions/rth_modified_std_module.php';
```

Comment:
This line is not required. The Stripe module has the std-module as a dependency which already handles auto loading for us.

Could you remove it please?